app/ante/evm/eth.go

```diff
@@ -301,14 +303,10 @@ func (ctd CanTransferDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate
 		}
 
 		baseFee := ctd.evmKeeper.GetBaseFee(ctx, ethCfg)
-
-		coreMsg, err := msgEthTx.AsMessage(signer, baseFee)
```

Comment:
as you already updated `msgEthTx.AsMessage` so imo should keep using it instead of this long and cause unnecessary change diff
```go
from := common.BytesToAddress(msgEthTx.GetFrom())
coreMsg := evmtypes.TxAsMessage(msgEthTx.AsTransaction(), baseFee, from)
```